requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+pytest==6.2.5
+black==21.11b1
+beautifulsoup4==4.10.0
+checkqc==3.6.6
+lxml==4.6.4
```

Comment:
If we used conda to set this up we could, since it can be downloaded via bioconda. To keep things simple, the instructions only includes how to set it up with virtualenv. It is expected that pre-requisites are installed to be able to run the integration tests.

Reply:
Fixed in https://github.com/Molmed/seqreports/pull/46/commits/5d508dde7009ebd22492d0c41599a61fdfa5c0bd